dao/src/main/java/greencity/repository/impl/EventSearchRepoImpl.java

```diff
@@ -170,7 +170,7 @@ private void addStatusesPredicate(List<EventStatus> eventStatuses, Long userId,
     private void addTagsPredicate(List<String> tags, Root<Event> eventRoot, List<Predicate> predicates) {
         if (tags != null && !tags.isEmpty()) {
             ListJoin<Tag, TagTranslation> tagsJoin = eventRoot.join(Event_.tags).join(Tag_.tagTranslations);
-            predicates.add(tagsJoin.get(TagTranslation_.NAME).in(tags));
+            predicates.add(criteriaBuilder.upper(tagsJoin.get(TagTranslation_.NAME)).in(tags));
```

Comment:
it is also a good idea to make tags be in upper case too

Reply:
Ok.